service/src/resources/hive-webapps/hiveserver2/logconf.jsp

```diff
@@ -132,7 +134,10 @@
 
                 <button id="log-level-submit" type="button" class="btn btn-primary">Submit</button>
             </form>
-
+            <% } else { %>
+                <p>Cannot configure logging rules unless you user <%= hiveSession.getUserName() %></p>
```

Comment:
Not quite understand what the sentence means? unless you are an admin user ?

Reply:
Yeah, it's a typo. The text should be "unless user user_name has admin privileges"